prime-router/metadata/organizations-local.yml

```diff
@@ -133,6 +134,7 @@
         jurisdictionalFilter: [ "matches(ordering_facility_state, PA)"]
         transforms: { deidentify: false }
         defaults:
+          processing_mode_code: T
           redox_destination_id: cd019512-8a58-40e5-a416-4e597404f9b8
```

Comment:
This element was defaulted to true  before. Now the default will change per environment.